keps/sig-storage/20200609-csi-service-account-token.md

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
The kep metadata format has changed too. The kep number is the enhancement issue number, and this metadata is in a different file. See https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/695-skip-permission-change as an example.

Reply:
done.